components/desktop/gexif/Makefile

```diff
@@ -27,9 +25,9 @@ COMPONENT_ARCHIVE_HASH=	\
     sha256:beed223c55fc32ba7bf2f91e69e6daf2087c8b44ccfd9191c34058bacd776c24
 COMPONENT_ARCHIVE_URL=	http://sourceforge.net/projects/libexif/files/gexif/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)/download
 
-include $(WS_MAKE_RULES)/prep.mk
-include $(WS_MAKE_RULES)/configure.mk
-include $(WS_MAKE_RULES)/ips.mk
+test:		$(NO_TESTS)
+
```

Comment:
This should be
TEST_TARGET= $(NO_TESTS)

Reply:
done